modules/post/multi/escalate/aws_create_iam_user.rb

```diff
@@ -0,0 +1,130 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'metasploit/framework/aws/client'
+
+class MetasploitModule < Msf::Post
+
+  include Metasploit::Framework::Aws::Client
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => "Create an AWS IAM User",
+        'Description'    => %q{
+          This module will attempt to create an AWS (Amazon Web Services) IAM
+          (Identity and Access Management) user with Admin privileges.
+        },
+        'License'        => MSF_LICENSE,
+        'Platform'       => %w(unix),
+        'SessionTypes'   => %w(shell meterpreter),
+        'Author'         => ['Javier Godinez <godinezj[at]gmail.com>'],
+        'References'     => [
+          [ 'URL', 'https://github.com/devsecops/bootcamp/raw/master/Week-6/slides/june-DSO-bootcamp-week-six-lesson-three.pdf' ]
+        ]
+      )
+    )
+
+    register_options(
+      [
+        OptString.new('IAM_USERNAME', [false, 'Name of the user to be created (leave empty or unset to use a random name)', '']),
```

Comment:
This is also the name of the group and the policy.  Might be worth clarifying this in the module and documentation and/or using a different name for this option.